static/app/views/dashboardsV2/widgetCard/widgetCardContextMenu.tsx

```diff
@@ -224,47 +232,53 @@ function WidgetCardContextMenu({
   return (
     <WidgetViewerContext.Consumer>
       {({setData}) => (
-        <ContextWrapper>
-          <Feature organization={organization} features={['dashboards-mep']}>
-            {isMetricsData === false && (
-              <SampledTag
-                tooltipText={t('This widget is only applicable to indexed events.')}
-              >
-                {t('Indexed')}
-              </SampledTag>
-            )}
-          </Feature>
-          <StyledDropdownMenuControl
-            items={menuOptions}
-            triggerProps={{
-              'aria-label': t('Widget actions'),
-              size: 'xs',
-              borderless: true,
-              showChevron: false,
-              icon: <IconEllipsis direction="down" size="sm" />,
-            }}
-            placement="bottom right"
-            disabledKeys={[...disabledKeys]}
-          />
-          {showWidgetViewerButton && (
-            <OpenWidgetViewerButton
-              aria-label={t('Open Widget Viewer')}
-              priority="link"
-              size="zero"
-              icon={<IconExpand size="xs" />}
-              onClick={() => {
-                setData({
-                  seriesData,
-                  tableData,
-                  pageLinks,
-                  totalIssuesCount,
-                  seriesResultsType,
-                });
-                openWidgetViewerPath(widget.id ?? index);
-              }}
-            />
+        <MEPConsumer>
+          {metricSettingContext => (
+            <ContextWrapper>
+              {(organization.features.includes('dashboards-mep') ||
+                organization.features.includes('mep-rollout-flag')) &&
+                isMetricsData === false &&
+                metricSettingContext &&
+                metricSettingContext.metricSettingState !== MEPState.transactionsOnly && (
```

Comment:
Above in `widgetCard/index.tsx`, you deconstruct `({metricSettingState})`, should we do that here to simplify? Not a big deal but this conditional is getting large

Reply:
I actually had it as deconstructed before but there are some scenarios where the arg is actually undefined and would error due to the deconstruction so I changed it to this. I should probably replace the other deconstructed {metricSettingState} as well